src/arrayMethodLastIndexOf.js

```diff
@@ -4,8 +4,22 @@
  * Implement method lastIndexOf
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex = this.length) {
+    if (searchElement === undefined) {
+      return -1;
+    }
+
+    let i = this.length + fromIndex;
+    if (fromIndex > 0) {
+      i = this.length;
```

Comment:
```suggestion
      i = fromIndex;
```
Если передать fromIndex=5, то в i присвоится this.length
Это значит что мы проверим весь массив, а не с 5 элемента до нулевого